test/fip.config

```diff
@@ -0,0 +1,7 @@
+image test.fip {
+	fip {
+		extraargs = "--align 64"
```

Comment:
Did you add the `--align` just to test this option, or is this something that is often needed? If it is needed then a separate option would be better.

Reply:
Just a test. I didn't have a use for it so far.